tiled/server/pydantic_array.py

```diff
@@ -60,14 +70,16 @@ def to_numpy_str(self):
         # so the reported itemsize is 4x the char count.  To get back to the string
         # we need to divide by 4.
         size = self.itemsize if self.kind != Kind.unicode else self.itemsize // 4
-        return f"{endianness}{self.kind.value}{size}"
+        units = f"[{self.units}]" if self.units else ""
+        return f"{endianness}{self.kind.value}{size}{units}"
 
     @classmethod
     def from_json(cls, structure):
         return cls(
             kind=Kind(structure["kind"]),
             itemsize=structure["itemsize"],
             endianness=Endianness(structure["endianness"]),
+            units=structure.get("units", "s"),
```

Comment:
defaulting to seconds?

Reply:
this must be a weird typo, as I was just copy-pasting this part `structures/array.py`. Changing it to empty string; thanks for catching this!

Reply:
Changed the default units to `None`, so this is not relevant anymore.